rpcs3/Input/pad_thread.cpp

```diff
@@ -272,7 +278,7 @@ void pad_thread::operator()()
 			{
 				while (thread_ctrl::state() != thread_state::aborting)
 				{
-					if (!pad::g_enabled || Emu.IsPaused() || !is_input_allowed())
+					if (!pad::g_enabled || !is_input_allowed())
```

Comment:
Even more here

Reply:
Maybe just also increase the sleep here if paused

Reply:
Fixed.